### DISCUSSION TITLE
Customize account theme menu items

### DISCUSSION THREAD

**Discussion by Luuk12345** [keycloak/keycloak#14957] [Q&A]
Hello,
I'm currently customizing my first keycloak theme and I'm trying to remove the 'Sessions' and 'Applications' menu items from the account theme. I cannot seem to find a way to do this. Could someone point me in the right direction? Thanks in advance. 
![keycloak vraag](https://user-images.githubusercontent.com/116069284/196385392-f4e89657-9298-4541-8ebd-6251aded243c.png)

**Comment by sschu** [comment]
You are using the old account console which is deprecated and will be removed soon. Please take a look at the new account console...

**Comment by Luuk12345** [comment]
Thank you @sschu, I will take a look into that. Do you know any way to disable the 'Sessions' and 'Applications' menu items in the meantime? We are not able to update to the new account console right away

**Reply by sschu** [reply]
You will most likely have to create your own theme. See https://www.keycloak.org/docs/latest/server_development/#creating-a-theme
